src/covid19sim/simulator.py

```diff
@@ -1222,8 +1222,8 @@ def at(self, location, city, duration):
                         h.contact_book.sent_messages_by_day[cur_day].append(h.cur_message(cur_day))
                         self.contact_book.sent_messages_by_day[cur_day].append(self.cur_message(cur_day))
 
-            t_overlap = (min(self.leaving_time, getattr(h, "leaving_time", self.env.ts_initial+SECONDS_PER_HOUR*TICK_MINUTE)) -
-                         max(self.start_time,   getattr(h, "start_time",   self.env.ts_initial+SECONDS_PER_HOUR*TICK_MINUTE)))/(SECONDS_PER_MINUTE*TICK_MINUTE)
+            t_overlap = (min(self.leaving_time, getattr(h, "leaving_time", self.env.ts_initial+SECONDS_PER_HOUR)) -
+                         max(self.start_time,   getattr(h, "start_time",   self.env.ts_initial+SECONDS_PER_HOUR))) / SECONDS_PER_MINUTE
```

Comment:
Why was this wrong earlier?

Reply:
This, plus lines 1179-1180, constitute the places where `start_time` and `leaving_time` are used. Both assumed that `duration` is in units of `TICK_MINUTE`. The timeout, on the other hand, assumed that `duration` was in units of minutes.

You should not try to delve too deeply into why the code, as it was before, makes sense. It *does not*. In #147, I had made all refactorings that exactly preserve the simulator's output, and this requires maintaining the `TICK_MINUTE` nonsense to reproduce bit-exactly the pre-change simulator output, bugs included. By deleting this code, we consistently treat `duration` as minutes.

In the future I will make further changes that standardize all time durations onto seconds, but **this is a breaking commit**. The simulator knobs **will need to be recalibrated**, because this bug caused infection probabilities to be overestimated.

Reply:
amazing. Thanks olexa.

Reply:
Understood!